### PR TITLE
multiple pages: rephrase without adjectives

### PR DESCRIPTION
**Pull request by schneiderl**
Hi
On multiple occasions ([1](https://github.com/tldr-pages/tldr/pull/3004#pullrequestreview-236040219) [2](https://github.com/tldr-pages/tldr/pull/3020#issuecomment-492260939) [3](https://github.com/tldr-pages/tldr/pull/1137#discussion_r85696118) ) I've seen us talking about omitting adjectives in our command descriptions. 
And, to be honest, it always bothered me a little. 

This PR rephrases a few pages to get rid of some adjectives that slipped in.

If this PR gets approved I intent to update other pages that also have adjectives on the descriptions.

### CODE REVIEW

File: pages/common/fd.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # fd
 
-> A simple, fast and user-friendly alternative to `find`.
+> A alternative to `find`.
```
Comment (andrik) [inline pages/common/fd.md]:
```suggestion
> An alternative to `find`.
```

File: pages/common/fd.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # fd
 
-> A simple, fast and user-friendly alternative to `find`.
+> An alternative to `find`.
```
Comment (sbrl) [inline pages/common/fd.md]: I guess removing them is a good thing to make the pages more impartial, but this now leaves me asking "why would I use it? What's different about it compared to `find`?"

File: pages/common/hugo.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # hugo
 
-> Hugo is a fast and flexible static site generator.
+> Hugo is a static site generator.
```
Comment (sbrl) [inline pages/common/hugo.md]: Again, this leaves me wondering about many aspects of Hugo. What about this?
```suggestion
> Template-based static site generator. Uses modules, components, and themes.
```

File: pages/common/ipfs.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # ipfs
 
 > Inter Planetary File System.
-> A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
+> A peer-to-peer hypermedia protocol.
```
Comment (sbrl) [inline pages/common/ipfs.md]: I think we can include maybe a few of these adjectives, if we qualify them correctly. What about this?
```suggestion
> A peer-to-peer hypermedia protocol. Aims to make the web more open.
```

File: pages/common/rg.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # ripgrep
 
-> A fast command-line search tool.
+> A command-line search tool.
```
Comment (sbrl) [inline pages/common/rg.md]: Another instance of adjective removal resulting in a lack of detail, in my opinion. What about this?
```suggestion
> A recursive line-oriented CLI search tool.
> Aims to be a faster alternative to `grep`.
```

File: pages/common/fd.md (hunk 2 above)
Comment (schneiderl) [reply pages/common/fd.md]: I see your concern and agree with all of your comments. It's a matter of rephrasing the descriptions properly. For this specific case, though, I have no idea of how to phrase it differently.

Comment (sbrl) [reply pages/common/fd.md]: Hrm. Taking a look at the README, what about this?
```suggestion
> An alternative to `find`.
> Aims to be faster and easier to use than `find`.
```

Comment (schneiderl) [reply pages/common/fd.md]: Looks good.

**Pull request merged**